alerts/openshift-container-storage-operator/CephNodeDown.md

```diff
@@ -0,0 +1,57 @@
+# CephNodeDown
+
+## Meaning
+
+This alert indicates that one of the Ceph storage node went down.
+
+## Impact
+
+A node running Ceph pods is down. While storage operations will continue to
+function as Ceph is designed to deal with a node failure, it is recommended
+to resolve the issue to minimise risk of another node going down and affecting
+storage functions.
+
+## Diagnosis
+
+The alert message will clearly indicate which node is down
+
+    Storage node <nod-name> went down. Please check the node immediately.
+
+## Mitigation
+
+Document the current OCS pods (running and failing):
+
+    oc -n openshift-storage get pods
+
+The OCS resource requirements must be met in order for the osd pods to be scheduled on the new node. This may take a few minutes as the ceph cluster recovers data for the failing but now recovering osd.
+
+To watch this recovery in action ensure the osd pods were actually placed on the new worker node.
+
+Check if the previous failing osd pods are now running:
+
+    oc -n openshift-storage get pods
+
+If the previously failing osd pods have not been scheduled, use describe and check events for reasons the pods were not rescheduled.
+
+Describe events for failing osd pod:
+
+    oc -n openshift-storage get pods | grep osd
+
+Find a failing osd pod(s):
+
+    oc -n openshift-storage describe pods/<osd podname from previous step>
+
+In the event section look for failure reasons, such as resources not being met.
+
+In addition, you may use the rook-ceph-toolbox to watch the recovery. This step is optional but can be helpful for large Ceph clusters.
+
+**Determine failed OCS Node** [determine_failed_ocs_node](helpers/determine_failed_ocs_node.md)
```

Comment:
we do not have the "determine_failed_ocs_node.md" .. you will need to add it to helpers

Reply:
I think this file is included in https://github.com/openshift/runbooks/pull/150 , but with other file name. Please replace with the right one.

Reply:
Yup, corrected